examples/ampcontext-creative.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+  <head>
+    <style>body { background: lightblue; }</style>
+    <script src="./ampcontext-creative.js"></script>
```

Comment:
I prefer editing them in separate files when I'm working, does it matter for example code? if so I can do it